lock-impl/build.gradle

```diff
@@ -7,4 +7,7 @@ dependencies {
   compile 'com.palantir.patches.sourceforge:trove3:' + libVersions.trove
   compile group: 'com.palantir.remoting1', name: 'tracing'
   compile group: 'joda-time', name: 'joda-time'
+
+  testCompile group: 'org.assertj', name: 'assertj-core'
+  testCompile group: 'org.mockito', name: 'mockito-core'
```

Comment:
We need to run `./gradlew generateLock saveLock` to update the version.lock files.